test/rules/padding-between-test-function.js

```diff
@@ -0,0 +1,258 @@
+                endColumn: 1
+            }, {
+                message: ALWAYS_MESSAGE,
+                line: 2,
+                column: 23,
+                endLine: 3,
+                endColumn: 1
+            } ],
+            output: [
+                'describe(function(){',
+                '',
+                'it("", function () {})',
+                '',
+                '})'
+            ].join('\n')
+        },
+        {
+            code: [
+                '[1,2,3].forEach(function () {',
+                '',
+                'it("", function () {})',
+                '',
+                '});'
+            ].join('\n'),
+            options: [ 'never' ],
+            errors: [ {
+                message: NEVER_MESSAGE,
+                line: 1,
+                column: 30,
+                endLine: 3,
+                endColumn: 1
+            }, {
+                message: NEVER_MESSAGE,
+                line: 3,
+                column: 23,
+                endLine: 5,
+                endColumn: 1
+            } ],
+            output: [
+                '[1,2,3].forEach(function () {',
+                'it("", function () {})',
+                '});'
+            ].join('\n')
+        },
+        {
+            code: [
+                '[1,2,3].forEach(function () {',
+                'it("", function () {})',
+                '});'
+            ].join('\n'),
+            options: [ 'always' ],
+            errors: [ {
+                message: ALWAYS_MESSAGE,
+                line: 1,
+                column: 30,
+                endLine: 2,
+                endColumn: 1
+            }, {
+                message: ALWAYS_MESSAGE,
+                line: 2,
+                column: 23,
+                endLine: 3,
+                endColumn: 1
+            } ],
+            output: [
+                '[1,2,3].forEach(function () {',
+                '',
+                'it("", function () {})',
+                '',
+                '});'
+            ].join('\n')
+        },
+        {
+            code: [
+                'describe(function(){',
+                'var someVar=null;',
+                'beforeEach(function(){someVar = "test"});',
+                'it("foo", function(){assert(someVar)});',
+                '})'
+            ].join('\n'),
+            options: [ 'always' ],
+            errors: [ {
+                message: ALWAYS_MESSAGE,
+                line: 2,
+                column: 18,
+                endLine: 3,
+                endColumn: 1
+            }, {
+                message: ALWAYS_MESSAGE,
+                line: 3,
+                column: 42,
+                endLine: 4,
+                endColumn: 1
+            }, {
+                message: ALWAYS_MESSAGE,
+                line: 4,
+                column: 40,
+                endLine: 5,
+                endColumn: 1
+            } ],
+            output: [
+                'describe(function(){',
+                'var someVar=null;',
```

Comment:
Is this really the expected behavior or should there be another blank line between `describe` and `var`?

Reply:
I think I got it now. I was confused for a moment and thought the padding of the `describe` block should also affect the inner part, but that doesn’t make sense after a second thought.